src/rvs_util.cpp

```diff
@@ -161,12 +161,11 @@ bool fetch_gpu_list(int hip_num_gpu_devices, map<int, uint16_t>& gpus_device_ind
                 amd_mcm_gpu_found = true;
             }
 	}
-	if (cur_gpu_selected) {
-		if(!mcm_check || !mcm_die ){
+	// excluding secondary die from test list, drops power reading substantially
```

Comment:
Comment shall be something like "Add both primary and secondary GPU die to the device list" and the current comment shall be a note in the comment section.